src/codec/cdp/tx.ts

```diff
@@ -6603,6 +6746,20 @@ export class MsgClientImpl implements Msg {
       )
     );
   }
+
+  ConvertTokenInCdpToGroupTokens(
+    request: MsgConvertTokenInCdpToGroupTokens
+  ): Promise<MsgConvertTokenInCdpToGroupTokensResponse> {
+    const data = MsgConvertTokenInCdpToGroupTokens.encode(request).finish();
+    const promise = this.rpc.request(
+      "Switcheo.carbon.cdp.Msg",
+      "ConvertTokenInCdpToGroupTokens",
+      data
+    );
+    return promise.then((data) =>
+      MsgConvertTokenInCdpToGroupTokensResponse.decode(new _m0.Reader(data))
+    );
+  }
```

Comment:
Here as well, run through this list of rpc txs and add module functions if they are not added yet.